_positions/Innovation-Specialist-GS-13.md

```diff
@@ -0,0 +1,88 @@
+Government, and working to align strategies and practice; conferring
+with key government/private officials and top experts in the field,
+representing the GSA at technical symposia and/or conferences;
+- Demonstrating thought leadership and contributing best practices in
+a variety of technologies and principles; and
+- Performing other related duties as assigned.
+
+###Qualifications Required
+This highly competitive program seeks talented, diverse individuals from the
+innovation community, and candidates are required to demonstrate
+expertise in a number of competencies. The position requires diverse
+knowledge, including:
+- Mastery knowledge of and expertise in a wide variety of
+technologies and or technical concepts to facilitate the delivery of
+services to consumers and businesses.
+- Comprehensive knowledge of and expertise in user centered
+design, consumer internet technologies, service delivery, and
+management approaches, allowing the incumbent to provide sound
+and authoritative leadership on all issues related to the program.
+- Expert knowledge and expertise in product delivery, agile
+development methodologies, and open source software
+development.
+- Knowledge of and expertise in driving and implementing technology
+solutions that overcome significant challenges resulting from
+complex or bureaucratic environments, or technically difficult
+problems.
+- Comprehensive knowledge of and expertise in all stages of product
+or business development, and ability to lead complex, technology
+and policy initiatives from inception to implementation.
+- Knowledge of and expertise in frameworks for evaluating a range of
+qualitative and quantitative metrics around user needs, resource
+prioritization, and program management.
+- Skill in oral communications to present sensitive recommendations
+to higher authority, to obtain compliance with policies from activities
+nationwide, to articulate positions/policy of vast technical complexity
+and to represent the agency on task forces.
+- Skill in analysis in order to apply a wide range of qualitative and/or
+quantitative methods for assessing and improving program
+effectiveness.
+
+Candidates are also evaluated for their competencies in key areas,
+including:
+
+- Analysis and Creative Problem Solving: Expertise determining user
+needs or identifying market opportunities and designing and
+deploying solutions;
+- Technology Acumen: Knowledge of standard and emergent web
+technologies;
+- Entrepreneurship and Product Management: experience delivering
+products or services from concept to market;
+- Approach and Methodology: Expertise applying frameworks that
+emphasize delivery, iteration, quantitative and qualitative learnings,
+and responsiveness;
+- Empathy and Emotional Intelligence: Experience in clientfacing
+roles, navigating bureaucracy, business development, or delivering
+services with a strong userfacing
+or consumeroriented
```

Comment:
Something about the copy-and-paste is losing hyphens and replacing them with a different character that's only visible when editing this markdown. To avoid the problem in the future I recommend copy-and-pasting into a plain text editor (i.e., one that doesn't add formatting) and copy-and-pasting again from there.